keel-ec2-plugin/src/test/kotlin/com/netflix/spinnaker/keel/ec2/resource/TestUtils.kt

```diff
@@ -26,14 +29,53 @@ import com.netflix.spinnaker.keel.core.parseMoniker
 import org.apache.commons.lang3.RandomStringUtils
 import com.netflix.spinnaker.keel.clouddriver.model.ServerGroup as ClouddriverServerGroup
 
+
+/**
+ * There are two possible types of payloads that can be returned from clouddriver
+ *
+ * The older launch configuration
+ * The newer launch templates
+ */
+enum class LaunchInfo {
+  LAUNCH_CONFIG,
+  LAUNCH_TEMPLATE
+}
+
 fun ServerGroup.toCloudDriverResponse(
```

Comment:
This is test helper code that has been updated so it can return the two different types of launch info payloads